### PR TITLE
Added package properties SDKType and NewSDK

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Added a wrapper method in common_tasks to parse package name, version and whether it is track 2 or not.

### CODE REVIEW

File: scripts/devops_tasks/common_tasks.py [hunk 1]
```diff
@@ -461,3 +462,9 @@ def get_installed_packages(paths = None):
     ws = WorkingSet(paths) if paths else working_set  
     return ["{0}=={1}".format(p.project_name, p.version) for p in ws]
 
+def get_package_properties(setup_py_path):
+    """Parse setup.py and return package details like package name, version, whether it's track1 or track2
+    """
+    pkgName, version, _, requires = parse_setup(setup_py_path)
+    is_track2_pkg = pkgName in TRACK2_IDENTIFIER_REQ_PACKAGES or any(map(lambda x: (parse_require(x)[0] in TRACK2_IDENTIFIER_REQ_PACKAGES), requires))
```
Comment (weshaggard) [inline scripts/devops_tasks/common_tasks.py:469]: I suggest calling this is_new_sdk.

File: scripts/devops_tasks/common_tasks.py [hunk 2]
```diff
@@ -32,6 +32,7 @@
 
 DEV_REQ_FILE = "dev_requirements.txt"
 NEW_DEV_REQ_FILE = "new_dev_requirements.txt"
+TRACK2_IDENTIFIER_REQ_PACKAGES = ["azure-core", "azure-mgmt-core"]
```
Comment (weshaggard) [inline scripts/devops_tasks/common_tasks.py:35]: Lets try avoid using the name "track2". Lets use something like NEW_REQ_PACKAGES

**Review by weshaggard — APPROVED** [review]

File: scripts/devops_tasks/common_tasks.py [hunk 3]
```diff
@@ -461,3 +462,9 @@ def get_installed_packages(paths = None):
     ws = WorkingSet(paths) if paths else working_set  
     return ["{0}=={1}".format(p.project_name, p.version) for p in ws]
 
+def get_package_properties(setup_py_path):
```
Comment (scbedd) [inline scripts/devops_tasks/common_tasks.py:465]: So it's a new SDK if the package is azure-core, azure-mgmt-core, or if any of the requirements are one of those? 

What?

**Review by scbedd — APPROVED** [review]

File: scripts/devops_tasks/common_tasks.py (hunk 3 above)
Comment (weshaggard) [reply scripts/devops_tasks/common_tasks.py:465]: For reference we are following https://github.com/Azure/azure-sdk/issues/1261#issuecomment-691327942. The idea is we needed a way to detect if a package is track 2 (aka new).

**Pull request merged**